lib/execution_engine2/sdk/EE2Status.py

```diff
@@ -540,12 +541,9 @@ def _send_exec_stats_to_catalog(self, job_id):
         log_exec_stats_params["user_id"] = job.user
         app_id = job_input.app_id
         if app_id:
-            # Note this will not work properly for app_ids incorrectly separated by a '.',
-            # which happens in some KBase code (which needs to be fixed at some point) -
-            # notably the narrative data download code, maybe more
-            # It's been this way for a long time, so leave for now
```

Comment:
will revert

Reply:
I can't reply to above comment for some reason. 

I believe that was one problem, because the app stats panel in the job browserr tracks successes and failures and I didnt see any other method that tracks it. We just weren't tracking failures as far as I can tell, but then how would the app stats get populated with failures if we didn't send them here?

Reply:
It's because it was a reply to an earlier comment, so they only allow you to respond in one place, up there ^

I guess I don't understand the problem we're trying to fix. Is it that the catalog isn't getting app stats at all (which is what I thought it was) or that the catalog isn't getting failure data? If it's just the latter then this seems like the fix, yeah.